ar4si/arm-cca.md

```diff
@@ -0,0 +1,91 @@
+ThomasF: I am having trouble understanding how the two semantics (i.e., boot loader(s) configuration vs ISA configuration) can co-exist with each other in the same TV claim.  It looks like the intention of AR4SI config as currently specified ("A Verifier has appraised an Attester's configuration, and is able to make conclusions regarding the exposure of known vulnerabilities") is more naturally mapped to the former (i.e., boot loader's) than the latter - mostly because I wouldn't know how to "make conclusions regarding the exposure of known vulnerabilities" based on the platform's config claim alone.
+SimonF: Agreed. I think the original doc misses the nuisance of HW configuration affecting this ability.
+ThomasF: yeah, that's a "very" gray area.  I was pondering whether making platform's config fall into the hardware.GENUINE_HARDWARE basket, but it's probably not a huge improvement.
+
+[^2]: SimonF: This is a bit tricky because the presence & config of a Memory Protection Engine is Impdef. It should show up in the same sys properties that are used to show RME so we could note that it's profile capable.  Remind me: is this setting bitwise or absolute - i.e. can we have both Encrypted + Isolated for this entry or does it have to be one or the other?
+ThomasF: it's absolute: basically, ENCRYPTED absorbs ISOLATED
+
+[^3]: SimonF: Any system known to be RME capable and configured can claim ISOLATED. We could just do this from platform config but potentially also ImplementationID
+ThomasF: OK, so this can be implied if (tv.instance_identity == TRUSTWORTHY_INSTANCE) && (platform.config & RME_CAPABLE) ?
+
+# Realm
+
+|Bucket|Notes|Claims|Applicability|
+| --- | --- | --- | --- |
+|Catch-all||VERIFIER_MALFUNCTION|any unexpected exceptions during the appraisal/verification flow|
+|||NO_CLAIM|This MAY be the value for the buckets that are not applicable (i.e., those marked as “N.A.”), if they are included.  Alternatively, if platform appraisal fails flat, this can be the value that all Realm buckets are reset to (meaning: since platform is not successfully appraised, we cannot say anything about Realm integrity)|
+|||UNEXPECTED_EVIDENCE|On failures coming from the Realm token decoder|
+|||CRYPTO_VALIDATION_FAILED|on failures coming from the Realm token verifier (not including the binding to the Platform token, which is dealt with in the instance identity category as UNRECOGNIZED_INSTANCE)|
+|Instance identity||TRUSTWORTHY_INSTANCE|If crypto verification (including chaining) completes correctly|
+|||UNTRUSTWORTHY_INSTANCE|N.A.|
+|||UNRECOGNIZED_INSTANCE|If crypto verification is OK but chaining fails|
+|Config||APPROVED_CONFIG|N.A.|
+|||NO_CONFIG_VULNS|N.A.|
+|||UNAVAIL_CONFIG_ELEMS|N.A.|
+|||UNSAFE_CONFIG|N.A.|
+|||UNSUPPORTABLE_CONFIG|N.A.|
+|Executables|It looks like an “UNRECOGNIZED_BOOT” is missing… but really the problem here is that this claim conflates run-time and boot-time.|APPROVED_BOOT|if RIM and PV (if provided) match|
```

Comment:
What is the value of `Executables` field if RIM and PV do not match? Would the value ever be `APPROVED_BOOT` in paractice, if the value of the field depends on the subsequent evaluation of REM?

I agree, there should really be a spearate claim for "boot"...

Reply:
> What is the value of Executables field if RIM and PV do not match?

There is none at the moment.  UNRECOGNIZED_BOOT would be it - if it existed.
OK, I will raise an issue in the AR4SI repo for that.
Until then, we have the negative codepoints space to use to define our new experimental semantics.